src/proxy/offline/proxy.js

```diff
@@ -134,17 +134,13 @@ export default _.defaults(
                      * performs navigation trough items of given test parameters according to action parameters
                      * doesent need active internet connection
                      * @param navigator - navigator helper used with this proxy
-                     * @param {Object} options - options to manage the navigation
                      * @param {Object} options.testContext - current test testContext dataset
                      * @param {Object} results - navigtion result output object
                      */
-                    var navigate = function(navigator, options, results) {
```

Comment:
I'm ok with the remival of the `init`. However, the testContext should be set each time, as it is dependent to the current item.

Reply:
fixed